commit a01e4fb69efeb7a803e6d3a272c62224d7df2494
Author: David Rosca <nowrep@gmail.com>
Date:   Sat Apr 8 10:40:27 2017 +0200

    Calendar: Use correct language for month and day names
    
    Apply fix for bug 353715 also on QML side.
    
    Differential Revision: https://phabricator.kde.org/D5345

diff --git a/src/declarativeimports/calendar/qml/DaysCalendar.qml b/src/declarativeimports/calendar/qml/DaysCalendar.qml
index 02b23b6e5..2b3ce446c 100644
--- a/src/declarativeimports/calendar/qml/DaysCalendar.qml
+++ b/src/declarativeimports/calendar/qml/DaysCalendar.qml
@@ -312,7 +312,7 @@ Item {
             Components.Label {
                 width: daysCalendar.cellWidth
                 height: daysCalendar.cellHeight
-                text: Qt.locale().dayName(calendarBackend.firstDayOfWeek + index, Locale.ShortFormat)
+                text: Qt.locale(Qt.locale().uiLanguages[0]).dayName(calendarBackend.firstDayOfWeek + index, Locale.ShortFormat)
                 font.pixelSize: Math.max(theme.smallestFont.pixelSize, daysCalendar.cellHeight / 3)
                 opacity: 0.4
                 horizontalAlignment: Text.AlignHCenter
diff --git a/src/declarativeimports/calendar/qml/MonthMenu.qml b/src/declarativeimports/calendar/qml/MonthMenu.qml
index aa5d3b46d..beddf6852 100644
--- a/src/declarativeimports/calendar/qml/MonthMenu.qml
+++ b/src/declarativeimports/calendar/qml/MonthMenu.qml
@@ -28,7 +28,7 @@ PlasmaComponents.Menu {
     property Instantiator items: Instantiator {
         model: 12
         PlasmaComponents.MenuItem {
-            text: capitalizeFirstLetter(Qt.locale().standaloneMonthName(index))
+            text: capitalizeFirstLetter(Qt.locale(Qt.locale().uiLanguages[0]).standaloneMonthName(index))
             onClicked: calendarBackend.displayedDate = new Date(year, index, 1)
         }
         onObjectAdded: testMenu.addMenuItem(object)
diff --git a/src/declarativeimports/calendar/qml/MonthView.qml b/src/declarativeimports/calendar/qml/MonthView.qml
index a2229e5f2..e3c0e95d0 100644
--- a/src/declarativeimports/calendar/qml/MonthView.qml
+++ b/src/declarativeimports/calendar/qml/MonthView.qml
@@ -143,7 +143,7 @@ PinchArea {
         Component.onCompleted: {
             for (var i = 0; i < 12; ++i) {
                 append({
-                    label: Qt.locale().standaloneMonthName(i, Locale.LongFormat),
+                    label: Qt.locale(Qt.locale().uiLanguages[0]).standaloneMonthName(i, Locale.LongFormat),
                     monthNumber: i + 1,
                     isCurrent: true
                 })
